internal-docs/design/designer-workflow.md

```diff
@@ -34,8 +34,6 @@ When you are starting new work in the Paste Figma libraries, be sure to:
 - First, create a branch off of the file you want to make changes to. Scroll to this section for [branch naming conventions](#figma-branching).
 - If you’re working with library updates from another library, pull those updates into a new branch rather than the main branch.
 - Put a “🚧” emoji at the start of the page name for any in-progress component (e.g. “🚧 Button”).
```

Comment:
**question:**  I'm not sure if we still have to do this lol I haven't been doing it.

Reply:
Nah, we don't do this anymore since Figma has the Dev status flags

Reply:
lol I forgot that we used to do this. It was more important when we had more than 2 people working in the file at the same time.